setup.py

```diff
@@ -47,7 +47,7 @@
         "vivarium_build_utils>2.0.3,<3.0.0",
         "gbd_mapping>=4.1.3",
         "vivarium>=3.4.5,<4.0.0",
-        "vivarium_public_health>=4.2.2,<5.0.0",
+        "vivarium_public_health>=4.3.10,<5.0.0",
```

Comment:
Needs 4.3.10 because that includes a bugfix to not drop stratifications from final results